processing/src/main/java/io/druid/segment/StringDimensionIndexer.java

```diff
@@ -534,79 +535,6 @@ public void fillBitmapsFromUnsortedEncodedArray(
     }
   }
 
-  @Override
-  public ValueMatcher makeIndexingValueMatcher(
-      final String matchValue,
-      final IncrementalIndexStorageAdapter.EntryHolder holder,
-      final int dimIndex
-  )
-  {
-    final int encodedVal = getEncodedValue(matchValue, false);
-    final boolean matchOnNull = Strings.isNullOrEmpty(matchValue);
-    if (encodedVal < 0 && !matchOnNull) {
-      return new BooleanValueMatcher(false);
-    }
-
-    return new ValueMatcher()
```

Comment:
@cheddar are you saying it would work to have two different types of value matchers in the same benchmark? We only have the one now (String) but we can definitely mix them together in the same benchmark when we add more.